Zonk_2.0/GAME_LOGIC.h

```diff
@@ -0,0 +1,330 @@
+int* createRandomDices(int);
+void printDices(int*, int);
+void playerMove(int*, int*, int*);
+void botMove(int*, int*, int*);
+void calculateTheSum(int*, int*);
+int playersRoundResult(int*, int*, int*);
+int botRoundResult(int*, int*, int*);
+
+
+/*Функция получает в качестве параметра итоговый результат. Если победа достигнута - возвращает истину, в противном случае - ложь*/
+bool checkWin(int result)
+{
+	return result > 4000 - 1;
+}
+
+
+/*Функия получает в качестве параметра количество костей и возвращает указатель на массив рандомных костей*/
```

Comment:
Your randomization logic is hard coded into function body (`*(dices + i) = rand() % 5 + 1;`). Look at _high order function_ topic.